tests/pytorch/test_nn.py

```diff
@@ -1515,20 +1559,6 @@ def test_hgt(idtype, in_size, num_heads):
     sorted_x = sorted_g.ndata['x']
     sorted_y = m(sorted_g, sorted_x, sorted_ntype, sorted_etype, presorted=False)
     assert sorted_y.shape == (g.num_nodes(), head_size * num_heads)
-    # mini-batch
```

Comment:
I think you need to revert the removal of the unit tests below.

Reply:
unit test reverted. Yes, got it.